experimental/direct_win/README.md

```
@@ -0,0 +1,90 @@
+
+#### 6. client-admin 側ライブラリのインストール
+
+```
+cd client-admin
+npm install
+cd ..
+```
+
+#### 7. server 側ライブラリのインストール（PDM + 仮想環境）
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)